data/lang/core/en.json

```diff
@@ -1851,6 +1919,18 @@
     "description": "Area unit: square meter (m²)",
     "message": "m²"
   },
+  "UNIT_TEMPERATURE_CELSIUS": {
+    "description": "Temperature unit: degrees Celsius (°C)",
+    "message": "°C"
+  },
+  "UNIT_TEMPERATURE_FAHRENHEIT": {
+    "description": "Temperature unit: Fahrenheit (F)",
+    "message": "F"
+  },
+  "UNIT_TEMPERATURE_KELVIN": {
+    "description": "Temperature unit: degrees Kelvin (°K)",
+    "message": "°K"
```

Comment:
°K -> K

Reply:
Fixed in 74f390aaeacdb1a5be8e34ca3d343e415f39b82d